plugins/ata/fu-ata-device.c

```diff
@@ -166,6 +166,136 @@ fu_ata_device_parse_id_maybe_dell (FuAtaDevice *self, const guint16 *buf)
 	guid_efi = fu_ata_device_get_guid_safe (buf, 129);
 	if (guid_efi != NULL)
 		fu_device_add_guid (FU_DEVICE (self), guid_efi);
+
+	/* owned by Dell */
+	fu_device_set_vendor (FU_DEVICE (self), "Dell");
+	fu_device_set_vendor_id (FU_DEVICE (self), "ATA:0x1028");
+}
+
+static void
+fu_ata_device_parse_vendor_name (FuAtaDevice *self, const gchar *name)
+{
+	struct {
+		const gchar	*prefix;			/* in CAPS */
+		guint16		 vid;
+		const gchar	*name;
+	} map_name[] = {
+		/* vendor matches */
+		{ "FUJITSU*",	0x10cf,	"Fujitsu" },
+		{ "SEAGATE*",	0x1bb1,	"Seagate" },
+		{ "ADATA*",	0x1cc1,	"ADATA" },
```

Comment:
This appears mostly sorted except for these

Reply:
Ohh yes, good catch. Thanks.